src/models/parties/parties.js

```diff
@@ -6,40 +6,42 @@ export const DEFAULT_PARTY_COUNT = 5;
 export const MAX_PARTY_COUNT = 10;
 
 export function makeParties() {
-  let names = Set([
-    'asparagus',
-    'corn',
-    'cabbage',
-    'turnip',
-    'radish',
-    'carrot',
-    'onion',
-    'spinach',
-    'mushroom',
-    'eggplant'
-  ]);
+  let names = [
+    {'asparagus': 'rgb(151, 190,17)',
```

Comment:
So what you have here is a _property_ with a _key_ of `'asparagus'` and a _value_ of `'rgb(151, 190, 17)'`. The issue is that objects properties should have predictable _keys_ and _values_. If the consumer of these objects wants to access the name or color of the party, it has no ability to do that because it wouldn't know which key to look under.

A good rule of thumb is that for an array of objects, all the objects should have roughly the same schema. This means that they should have the same _keys_ and each _key_ should have a consistent `type`.

A way to resolve this would be to have a `color` property and a `name` property and to store `'asparagus'` and  `'rgb(151, 190, 17)'` in those properties, respectively.